docs/deployment-process/execution-containers-for-workers/index.md

```diff
@@ -1,25 +1,55 @@
 ---
 title: Execution containers for workers
-description: How to set a step in your deployment process to run inside a container.
+description: How to set a step in your deployment/runbook process to run inside a container.
 position: 15
 ---
 
-For a [step](/docs/deployment-process/steps/index.md) running on a [worker](/docs/infrastructure/workers/index.md) or on [Octopus server](/docs/infrastructure/workers/built-in-worker.md) that you define in your [deployment processes](/docs/deployment-process/index.md), you can select a container for the step's execution. 
+For a [step](/docs/deployment-process/steps/index.md) running on a [worker](docs/infrastructure/workers/index.md) or on [Octopus Server](docs/infrastructure/workers/built-in-worker.md) that you define in your [deployment processes](/docs/deployment-process/index.md), you can select a container for the step's execution.
 
-## Minimum requirements for your step to run in a container
-The worker will need Docker installed. 
+## Minimum requirements
+You need Docker installed and running on the worker, in order to use execution containers for workers
 
-## Setup a container registry and an image 
 
-- Set a [feed](/docs/packaging-applications/package-repositories/docker-registries/index.md) in Octopus Deploy for a Docker registry. 
-- Add a project and define a deployment process. 
-- Select a worker pool. 
-- Select to run on a container for worker. 
-- Choose the previously added container registry. 
-- Enter an image name if already known. (e.g. !docker-image <octopusdeploy/worker-tools:ubuntu.18.04>)
-- Save.
-- Create release & deploy.
+## How to use execution containers for workers 
+
+- Configure a [feed](/docs/packaging-applications/package-repositories/docker-registries/index.md) in Octopus Deploy for a Docker registry.
+  - [Add Docker Hub as an external feed](https://octopus.com/blog/build-a-real-world-docker-cicd-pipeline#add-docker-hub-as-an-external-feed)
+- Add a project and define a deployment process.
+- Set the `Execution Location` for your step to `Run on a worker`.
+- In `Container Image` select `Runs on a worker inside a container`.
+- Choose the previously added container registry.
+- Enter an image name if already known. (e.g. !docker-image <octopusdeploy/worker-tools:1.0.1-ubuntu.18.04>).
```

Comment:
```suggestion
- Enter an image name if already known. (e.g. !docker-image <octopusdeploy/worker-tools:ubuntu.18.04>).
```

Sorry I thought I already made this change and linked to the docs that explain. No version number should be included here - the idea of the `!docker-include` is that it will keep the version up to date for us automatically.

Reply:
Ah right, I thought it would be best to show what a version number on the image name would look like to someone reading the docs.

Reply:
That’s what this edit will do. Suppose you have an image repo `foo-registry/bar`. You can either hard-code a value:

> foo-registry/bar:x.y.z-metadata

or use the `!docker-include` feature to keep the version number up to date automatically:

> !docker-include<foo-registry/bar:metadata>

Say the latest version of `metadata` is `3.2.1` when the docs are built - then that would compile to:

> foo-registry/bar:3.2.1-metadata

Here’s the docs: https://octopusdeploy.github.io/OctoStyle/markdown#referencing-docker-images

Reply:
Oh right!! sorry wasn't aware of that. I thought the octofront PR was just to validate that image exists.